scripts/PublishModules.ps1

```diff
@@ -0,0 +1,45 @@
+[CmdletBinding(SupportsShouldProcess=$true)]
```

Comment:
This script is pretty self-evident, but I'll copy my comment here.

It's not required, but it might be helpful going forward if we add to the top of the file comments providing a high-level description of the script.